src/components/Graph/PrimaryChart/PrimaryChart.tsx

```diff
@@ -21,6 +20,28 @@ import { colors } from 'theme/colors'
 import { MaxPrice } from '../MaxPrice'
 import { MinPrice } from '../MinPrice'
 
+const LinearGradient = lazy(() =>
+  import('@visx/gradient').then(({ LinearGradient }) => ({ default: LinearGradient })),
+)
+const ScaleSVG = lazy(() =>
+  import('@visx/responsive').then(({ ScaleSVG }) => ({ default: ScaleSVG })),
+)
+
+type AnimatedAreaSeriesType = React.ComponentType<
```

Comment:
This and `TooltipType` below were previously properly inferred, but now require types to be re-declared or we'll have some pretty nasty type errors WRT upstream `Datum extends object` and `HistoryData` not extending the old, outdated TS `object`